CONTRIBUTING.md

```diff
@@ -32,6 +32,42 @@ To get started we recommend:
 
 6. Install [golangci-lint] linter and run `make lint` to execute additional code linters.
 
+### Quick Start Local Development (MacOS)
+
+First install [colima](https://github.com/abiosoft/colima):
```

Comment:
I'm not sure if we should be solving the "get a local kubernetes" problem in the contributing documentation for troubleshoot. I'm also not familiar with this project and I would be curious if we were to make recommendations down to this level why we pick this over the dozens of other options. In particular, it would seem [kind](https://kind.sigs.k8s.io/) being an upstream sig is a fairly strong option.